modules/cert-manager-uninstall-console.adoc

```diff
@@ -22,3 +22,9 @@ You can uninstall the {cert-manager-operator} by using the web console.
 .. Navigate to *Operators* -> *Installed Operators*.
 .. Click the Options menu {kebab} next to the *{cert-manager-operator}* entry and click *Uninstall Operator*.
 .. In the confirmation dialog, click *Uninstall*.
+.. Remove the {cert-manager-operator} by running the following command:
++
+[source, terminal]
```

Comment:
This is only relevant if the user had an older version of cert-manager-operator TechPreview installed on their cluster i.e. cert-manager-operator.v1.7 or below.

In case, the OpenShift cluster started off directly with our GA operators this step would fail due to the absence of the ClusterOperator.

We need some means to factor in this information/insight.

Reply:
Please let me know if there is alternative for cert-manager v1.10 and above.

Reply:
For v1.10+ this step is not required at all 😅
So only old TechPreview users require that step at removal.